<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigRadioGroupBox</class>
 <widget class="QGroupBox" name="ConfigRadioGroupBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>146</width>
    <height>475</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>146</width>
    <height>475</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>146</width>
    <height>475</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>146</width>
    <height>475</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">GroupBox</string>
  </property>
  <property name="title">
   <string>GroupBox</string>
  </property>
  <property name="checkable">
   <bool>true</bool>
  </property>
  <property name="checked">
   <bool>false</bool>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>110</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QDial" name="dial_in_lo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>50</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimum">
    <number>60</number>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="singleStep">
    <number>10</number>
   </property>
   <property name="pageStep">
    <number>100</number>
   </property>
   <property name="value">
    <number>60</number>
   </property>
   <property name="notchTarget">
    <double>9.699999999999999</double>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDial" name="dial_in_hi">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>50</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1000</number>
   </property>
   <property name="maximum">
    <number>6000</number>
   </property>
   <property name="singleStep">
    <number>10</number>
   </property>
   <property name="pageStep">
    <number>100</number>
   </property>
   <property name="notchTarget">
    <double>9.699999999999999</double>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_bp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="text">
    <string notr="true">Frequency Range In</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_in_hi">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>54</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string notr="true"/>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="minimum">
    <double>1000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>6000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_in_lo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>54</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="minimum">
    <double>60.000000000000000</double>
   </property>
   <property name="maximum">
    <double>999.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>10.000000000000000</double>
   </property>
   <property name="value">
    <double>60.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_dest">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Destruction</string>
   </property>
  </widget>
  <widget class="QDial" name="dial_destr">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>247</y>
     <width>71</width>
     <height>91</height>
    </rect>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="notchTarget">
    <double>9.699999999999999</double>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_destr">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>332</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="maximum">
    <double>20.000000000000000</double>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>220</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_rm">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Ring Modulator</string>
   </property>
  </widget>
  <widget class="QDial" name="dial_rm">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>50</width>
     <height>64</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Modulator Frequency</string>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
   <property name="singleStep">
    <number>10</number>
   </property>
   <property name="pageStep">
    <number>100</number>
   </property>
   <property name="notchTarget">
    <double>9.699999999999999</double>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_rm">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="suffix">
    <string notr="true"> Hz</string>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>100.000000000000000</double>
   </property>
  </widget>
  <widget class="QDial" name="dial_rm_mix">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>50</width>
     <height>64</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Mix</string>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>10</number>
   </property>
   <property name="pageStep">
    <number>100</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="notchTarget">
    <double>9.699999999999999</double>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_rm_mix">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>54</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string notr="true"/>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>350</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_bp_out">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string notr="true">Frequency Range Out</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_out_lo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>54</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="minimum">
    <double>60.000000000000000</double>
   </property>
   <property name="maximum">
    <double>999.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>10.000000000000000</double>
   </property>
   <property name="value">
    <double>60.000000000000000</double>
   </property>
  </widget>
  <widget class="QDial" name="dial_out_lo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>50</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimum">
    <number>60</number>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="singleStep">
    <number>10</number>
   </property>
   <property name="pageStep">
    <number>100</number>
   </property>
   <property name="value">
    <number>60</number>
   </property>
   <property name="notchTarget">
    <double>9.699999999999999</double>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDial" name="dial_out_hi">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>380</y>
     <width>50</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1000</number>
   </property>
   <property name="maximum">
    <number>6000</number>
   </property>
   <property name="singleStep">
    <number>10</number>
   </property>
   <property name="pageStep">
    <number>100</number>
   </property>
   <property name="notchTarget">
    <double>9.699999999999999</double>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_out_hi">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>440</y>
     <width>54</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string notr="true"/>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="minimum">
    <double>1000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>6000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>10.000000000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
